config/locales/en/claims/support/schools.yml

```diff
@@ -28,6 +28,7 @@ en:
           title_with_error: "Error: Enter a school name, URN or postcode"
         check:
           add_organisation: Add organisation
+          save_organisation: Save organisation
```

Comment:
Do we still use `add_organisation`?

Reply:
Yes, it's used in the context of the page

![Screenshot from 2024-05-07 16-19-52](https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/9c1a264a-0694-49fc-9827-f275ec9cab3b)